iOS/docs/kickoff/README.md

```diff
@@ -146,6 +146,25 @@ And commit them:
 
 To configure continuous integration follow the guide [CI configuration for iOS projects](./configure-ci/README.md)
 
+### Configure Crash logger
+
+The project integrates [Rollbar](https://github.com/rollbar/rollbar-ios) as default crash logger.
+
+In order to enable it, ask a technical leader to create a new group and a new iOS project.
+
+Once created, sign in to [Rollbar](https://rollbar.com), and go to your project "Settings"/"Access Tokens".
+
+Then copy into all the `.xcconfig` files the following content:
+
+```
+ROLLBAR_CLIENT_ACCESS_TOKEN = ...
+ROLLBAR_SERVER_ACCESS_TOKEN = ...
+```
+
+Where `ROLLBAR_CLIENT_ACCESS_TOKEN` is the token under "post_client_item", and `ROLLBAR_SERVER_ACCESS_TOKEN` is the token under "post_server_item".
+
```

Comment:
Another possibility is writing:
```
ROLLBAR_CLIENT_ACCESS_TOKEN = <token under "post_client_item">
ROLLBAR_SERVER_ACCESS_TOKEN = <token under "post_server_item">
```

Don't know what is better. Any is good for me.